pepper-apis/api-build.sh

```diff
@@ -220,6 +220,64 @@ function docker_nginx_push {
     docker push "broadinstitute/pepper-nginx:$tag"
 }
 
+function print_usage() {
+    echo 'usage: api-build.sh <version> <environment> <dir> <option>...'
+    echo '       api-build.sh [-h, --help]'
+}
+
+function print_help() {
+    cat << 'EOM'
+A script to help automate a few different steps of the process
+for both local development and in CI deployments.
+
+Configurations are rendered into the directories `output-config`
+and `output-build-config`. Docker images, such as
+`broadinstitute/pepper-api-backend`, will be tagged with `$VER_ENV`.
+
+USAGE:
+  api-build.sh <VER> <ENV> <DIR> [OPTIONS...]
+
+VER
+  is the version of the code, should always be `v1`
+
+ENV
+  is the environment against which to build, one of
+  `dev`, `test`, `staging`, or `prod`
+
+DIR
+  is the local directory from whence `docker-compose` volumes originate,
+  use `.` for local builds, `/app` for CI builds
+
+OPTIONS:
+  --config          render local configurations
+  --docker-build    build docker images
+  --docker-push     build docker images and push to dockerhub
+  -h, --help        print this help message
+  --jenkins         runs CI workflow of build, test, and push docker images,
+                    should only use this in CI environment
+  --jenkins-test    only build docker images and run test suite
+  --local-deploy    build and test docker images for local development
+  --nginx           build and push docker image for nginx
+  --test            build and test docker images locally
+EOM
+}
+
+###
+#
+# Main
+
+if (( $# == 1 )); then
```

Comment:
Why is the `((` construct being used here instead of `[[`?

Reply:
I think people recommend using `((` instead of `[[` when doing numeric comparisons in bash?